src/schema/v2/ArtworkImport/artworkImport.ts

```diff
@@ -192,6 +193,10 @@ const ArtworkImportRowType = new GraphQLObjectType({
               resolve: ({ CertificateOfAuthenticity }) =>
                 CertificateOfAuthenticity,
             },
+            signatures: {
+              type: new GraphQLList(GraphQLString),
+              resolve: ({ Signature }) => Signature,
```

Comment:
Shouldn't this be `Signatures` instead, because it's a list of signatures?

```suggestion
              resolve: ({ Signatures }) => Signatures,
```

Reply:
It corresponds to the `normalized_name` in https://github.com/artsy/gravity/pull/18910 so it has to match that

Reply:
Should we call the schema field `signature` in that case as well? We should try to stick 1-1 whenever we can (especially for new things).